java/vector/src/main/java/org/apache/arrow/vector/VarBinaryVector.java

```diff
@@ -279,6 +281,15 @@ public TransferPair makeTransferPair(ValueVector to) {
     return new TransferImpl((VarBinaryVector) to);
   }
 
+  @Override
+  public ByteBuffer getByteBuffer(int index) {
+    if (isNull(index)) {
+      return null;
+    } else {
+      return ByteBuffer.wrap(getObject(index));
```

Comment:
is it possible to wrap the underlying data directly?

Reply:
Hi Micah, do you mean create a wrapper class to hold byte[] like [ByteArrayWrapper](https://github.com/apache/arrow/blob/2575a2b2ff515f4989cfeba4f7e3a6625881d8c5/java/vector/src/main/java/org/apache/arrow/vector/dictionary/ByteArrayWrapper.java)?